app/controllers/concerns/foreman/controller/discovered_extensions.rb

```diff
@@ -69,32 +69,27 @@ def perform_auto_provision original_host, rule
   end
 
   def perform_reboot_all hosts = Host::Discovered.all
-    result = true
-    error_message = _("Errors during reboot: %s")
-    overall_errors = ""
+    overall_errors = []
 
     if hosts.count > 0
       hosts.each do |discovered_host|
         begin
           unless discovered_host.reboot
-            error =  "#{discovered_host.name}: failed to reboot "
-            overall_errors << error
-            logger.error error
+            msg = "#{discovered_host.name}: Failed to reboot "
+            e = Excpetion.New()
+            overall_errors << WrappedException.new(e, msg)
```

Comment:
Do not use WrappedException if you don't have any exception to wrap. Just use Foreman::Exception and give it some text, then you won't see this twice as well.